web/src/test/java/com/epam/rft/atsy/cucumber/settings/channels/ChannelsStepDefs.java

```diff
@@ -135,8 +135,9 @@ public void the_title_field_filled() throws Throwable {
 
   @Then("^the list of channels changed$")
   public void the_list_of_channels_changed() throws Throwable {
+    waitChannelTableUntilPresentChannelName(channelName);
```

Comment:
This line is enough on its own to do the job.
However, for better readability and to comply with "the_new_channel_appears_in_the_list_of_channels()" I kept the next assertion.